impedance/plotting.py

```diff
@@ -73,3 +74,196 @@ def plot_nyquist(ax, Z, scale=1, units='Ohms', fmt='.-', **kwargs):
     t.set_size(18)
 
     return ax
+
+
+def plot_bode(axes, f, Z, scale=1, units='Ohms', fmt='.-', **kwargs):
+    """ Plots impedance as a Bode plot using matplotlib
+
+        Parameters
+        ----------
+        axes: list of 2 matplotlib.axes.Axes
+            axes on which to plot the bode plot
+        f: np.array of floats
+            frequencies
+        Z: np.array of complex numbers
+            impedance data
+        scale: float
+            scale for the magnitude y-axis
+        units: string
+            units for :math:`|Z(\\omega)|`
+        fmt: string
+            format string passed to matplotlib (e.g. '.-' or 'o')
+
+        Other Parameters
+        ----------------
+        **kwargs : `matplotlib.pyplot.Line2D` properties, optional
+            Used to specify line properties like linewidth, line color,
+            marker color, and line labels.
+
+        Returns
+        -------
+        ax: matplotlib.axes.Axes
+    """
+
+    ax_mag, ax_phs = axes
+
+    ax_mag.plot(f, np.abs(Z), fmt, **kwargs)
+    ax_phs.plot(f, -np.angle(Z), fmt, **kwargs)
```

Comment:
I think we want the phase to show in degrees, not radians.

`ax_phs.plot(f, -np.angle(Z, deg=True), fmt, **kwargs)`

Reply:
Good catch :) I'll fix this with a new commit